src/zarr_benchmarks/read_write_zarr/read_write_zarr_python_v2.py

```diff
@@ -30,6 +30,7 @@ def write_zarr_array(
     overwrite: bool,
     chunks: tuple[int],
     compressor: numcodecs.abc.Codec | None,
+    zarr_spec: Literal[2, 3],
```

Comment:
```suggestion
    zarr_spec: Literal[2],
```
Zarr-python v2 only supports zarr-spec 2